specs/interop/predeploys.md

```diff
@@ -117,6 +129,7 @@ A simple implementation of the `executeMessage` function is included below.
 function executeMessage(Identifier calldata _id, address _target, bytes calldata _msg) public payable {
     require(_id.timestamp <= block.timestamp);
     require(L1Block.isInDependencySet(_id.chainid));
+    require(_id.timestamp > L1Block.interopStart());
```

Comment:
We could choose to put this value directly in the CrossL2Inbox to save a call, what do you think?

Reply:
Yeah, I think its worth the gas savings

Reply:
Interestingly, this already exists in Crossl2Inbox and not L1Block, it's my mistake in the reference implementation